livy/src/main/resources/interpreter-setting.json

```diff
@@ -118,6 +118,11 @@
         "defaultValue": "1000",
         "description": "Max number of Spark SQL result to display."
       },
+      "zeppelin.livy.spark.sql.truncate": {
+        "propertyName": "zeppelin.livy.spark.sql.truncate",
+        "defaultValue": "true",
+        "description": "If true, truncate strings greater than 20 characters."
```

Comment:
I tired this, but I get the following error:  " found   : Int(20)\n"," required: Boolean\n".